src/WireMock.Net.Abstractions/Admin/Mappings/XmlNamespace.cs

```diff
@@ -0,0 +1,21 @@
+namespace WireMock.Admin.Mappings;
+
+/// <summary>
+/// Defines an xml namespace consisting of prefix and uri.
+/// <example>xmlns:i="http://www.w3.org/2001/XMLSchema-instance"</example>
+/// </summary>
+[FluentBuilder.AutoGenerateBuilder]
+public class XmlNamespace
+{
+    /// <summary>
+    /// The prefix.
+    /// <example>i</example>
+    /// </summary>
+    public string Prefix { get; set; }
```

Comment:
annotate with ` = null!;` and assign value `string.Empty`

Reply:
= null!; -> Done
-> where to put string.Empty?